### PR TITLE
Corrected continuation sequence number in event returned by subscribe…

### PR DESCRIPTION
**Pull request by daryl-d**
# Changes
- based off https://github.com/localstack/localstack/issues/4120

### CODE REVIEW

File: tests/integration/test_kinesis.py [hunk 1]
```diff
@@ -89,6 +90,9 @@ def test_subscribe_to_shard(self):
         results = []
         for entry in stream:
             records = entry['SubscribeToShardEvent']['Records']
+            continuation_sequence_number = entry['SubscribeToShardEvent']['ContinuationSequenceNumber']
+            # https://docs.aws.amazon.com/kinesis/latest/APIReference/API_SubscribeToShardEvent.html
+            self.assertTrue(re.fullmatch('^0|([1-9][0-9]{0,128})$', continuation_sequence_number) is not None)
```
Comment (whummer) [inline tests/integration/test_kinesis.py:95]: nitpick: could be changed to either:
```
self.assertIsNotNone(re.fullmatch('^0|([1-9][0-9]{0,128})$', continuation_sequence_number))
```
or:
```
self.assertTrue(re.fullmatch('^0|([1-9][0-9]{0,128})$', continuation_sequence_number))
```

**Review by whummer — COMMENTED** [review]
Thanks for this PR @daryl-d ! 👍 Only added a minor nitpick comment. Also - we've been facing build issues on `master`, which should now be resolved. Can you please rebase your branch onto latest `master` , to retrigger the build? Thanks!

File: tests/integration/test_kinesis.py (hunk 1 above)
Comment (daryl-d) [reply tests/integration/test_kinesis.py:95]: Sweet, I'll change this. First time I've touched python in quite a while :)

**Comment by daryl-d** [comment]
No worries @whummer, love using this project, glad I could make a small contribution :)
Code should be updated now

**Pull request merged**